guides/_subsections/view-targets-with-topology.md

```diff
@@ -0,0 +1,112 @@
+## [Use Topology View](#use-topology-view)
+
+**Topology View** provides a visual presentation of the all discovered JVM applications and all resources that are associated with them. It also allows users to perform actions on one or multiple targets.
```

Comment:
I think this should be italicized to match with other guides instead of bolded. Also should we consider calling it `The` Topology View ?